podcastparser.py

```diff
@@ -307,6 +313,31 @@ def start(self, handler, attrs):
                 chapter[optional] = value
 
         handler.get_episode_attr('chapters').append(chapter)
+        
+        
+class EpisodePersonAttr(Target):
+    WANT_TEXT = True
+         
+    def start(self, handler, attrs):
+        if not handler.get_episode_attr("people"):
+            handler.add_episode_people()
```

Comment:
Since the field is `person`, `persons` might be better, even if in English it'd probably be `people`?

Reason is that e.g. somebody searching the docs/source for "person" will find "persons", but not "people".

Reply:
Agreed. I've made this change.